org.ghidra_sre.Desktop.json

```diff
@@ -0,0 +1,72 @@
+{
+    "app-id" : "org.ghidra_sre.Desktop",
+    "default-branch": "stable",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "20.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.openjdk11"
+    ],
+    "command" : "ghidra",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--filesystem=home",
+        "--env=PATH=/usr/bin:/app/bin:/app/jdk/bin",
+        "--env=JAVA_HOME=/app/jdk"
+    ],
+    "modules" : [
+        {
+            "name": "openjdk",
+            "buildsystem": "simple",
+            "build-commands": [
+              "/usr/lib/sdk/openjdk11/installjdk.sh"
+            ]
+        },
+        {
+            "name": "icoutils",
+            "cleanup": [
+                "*"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://savannah.nongnu.org/download/icoutils/icoutils-0.31.1.tar.bz2",
+                    "sha256": "26e29d3c78f25d4cdf402501ac0414c51a9a092daebf6c9dee3b837dee693093"
+                }
+            ]
+        },
+        {
+            "name": "ghidra",
+            "buildsystem": "simple",
+            "build-commands": [
+                "for i in docs Extensions Ghidra GPL licenses server support ; do install -d $i /app/lib/ghidra/ ; cp -r $i/ /app/lib/ghidra/$i/ ; done",
+                "install -Dm755 ghidraRun /app/lib/ghidra/ghidraRun",
+                "icotool -x support/ghidra.ico --index=8",
+                "install -Dm644 ghidra_8_256x256x32.png /app/share/icons/org.ghidra_sre.Desktop.png",
+                "install -Dm644 org.ghidra_sre.Desktop.desktop /app/share/applications/org.ghidra_sre.Desktop.desktop",
+                "install -Dm644 org.ghidra_sre.Desktop.metainfo.xml /app/share/metainfo/org.ghidra_sre.Desktop.metainfo.xml",
+                "sed -i 's,bg,fg,' /app/lib/ghidra/ghidraRun",
+                "ln -s /app/lib/ghidra/ghidraRun /app/bin/ghidra"
+            ],
+            "sources" : [
+                {
+                    "type": "archive",
+                    "sha256": "8cf8806dd5b8b7c7826f04fad8b86fc7e07ea380eae497f3035f8c974de72cf8",
+                    "url": "https://ghidra-sre.org/ghidra_9.2.2_PUBLIC_20201229.zip"
+                },
+                {
+                    "type": "file",
+                    "path": "org.ghidra_sre.Desktop.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "org.ghidra_sre.Desktop.metainfo.xml"
```

Comment:
You know the drill, ideally these two should be upstreamed.

Reply:
.desktop file:
https://github.com/NationalSecurityAgency/ghidra/pull/1556

I planned on upstreaming the metainfo file once that got in.

Reply:
https://github.com/NationalSecurityAgency/ghidra/issues/2739